source/agora/network/Manager.d

```diff
@@ -142,6 +142,7 @@ public class NetworkManager
             bool delegate (in Address address) onFailedRequest)
             @safe pure nothrow @nogc
         {
+            assert(address !is Address.init);
```

Comment:
[RPC uses](https://github.com/bosagora/agora/blob/c795379eff36548751843c196a303f2d04f0457e/source/agora/network/Manager.d#L1390) `Address.init`, discussed this a little with @omerfirmak yesterday.

Reply:
Those are used for the incoming connections where we don't know what address server is reachable on.

Reply:
So we should not ban or whitelist these?

Reply:
We should not. Instead of banning them, we should probably remove that peer from the peer list,